docs/cosmos-hub/gaiacli.md

```diff
@@ -187,13 +187,12 @@ When you query an account balance with zero tokens, you will get this error: `No
 The following command could be used to send coins from one account to another:
 
 ```bash
-gaiacli tx send <destination_cosmos> 10faucetToken \
-  --chain-id=<chain_id> \
-  --from=<key_name> \
+gaiacli tx send <key_name> <destination_cosmos> 10faucetToken \
```

Comment:
Can this be changed to `<sender_address_or_key> <recipient_addr> <amount>`?

Also, add a `tip` that states when `--generate-only` is provided, the sender must be an address.